protect_archiver/cli/download.py

```diff
@@ -1,111 +1,152 @@
 import click
 
 from datetime import datetime
-from os import path
 
 from .base import cli
 from ..client import ProtectClient, ProtectError
 
 
 @cli.command("download", help="Download footage from a local UniFi Protect system")
-@click.argument("dest")
-@click.option("--address", default="unifi", help="CloudKey IP address or hostname")
-@click.option("--port", default=7443, help="UniFi Protect service port")
+@click.argument(
+    "dest",
+    type=click.Path(exists=True, writable=True, resolve_path=True)
+)
+@click.option(
```

Comment:
did you not want to keep the default?

I'm assuming for almost everyone its `unifi`

Reply:
`unifi` didn't resolve to an IP within my network when I tried it - turns out my DNS server wants a full local domain name for it to work. Re-added it as the default and removed the interactive prompt.